.eslintrc.js

```diff
@@ -1,20 +1,17 @@
 module.exports = {
-  'extends':
-            [
-              'eslint:recommended',
-              'google',
-            ],
-  'env': {
-    'browser': true,
-    'es2021': true,
-    'es6': true,
-    'mocha': true,
+  ignorePatterns: ['test/Core.spec.ts'],
```

Comment:
Doing this for now because of:
/home/runner/work/slide-starter/slide-starter/test/Core.spec.ts
1:1 error Parsing error: 'import' and 'export' may appear only with 'sourceType: module'

And doing that has other issues.